extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -319,7 +319,11 @@ export class ProjectsController {
 		telemetryProps.totalDuration = (actionEndTime - buildStartTime).toString();
 
 		const currentDeployIndex = this.deployInfo.findIndex(d => d.startDate === currentDeployTimeInfo);
-		this.deployInfo[currentDeployIndex].status = Status.success;
+		if (result.success === true) {
```

Comment:
can be simplified to `this.deployInfo[currentDeployIndex].status = result.success ? Status.success : Status.failed`